device.go

```diff
@@ -223,3 +252,16 @@ func NewDevice(path, devType string, major, minor int64, fileMode *os.FileMode,
 	device := createDevice(devInfo)
 	return device, nil
 }
+
+// bdf returns the BDF of pci device
+// Expected input strng format is [<domain>]:[<bus>][<slot>].[<func>] eg. 0000:02:10.0
+func bdf(deviceSysStr string) (string, error) {
```

Comment:
I would prefer `getDevBDF()` or `getBDF()` here since I think `bdf()` is a bit short and confusing for the reading of the code.

Reply:
ok. I can make that change.